doc/manual/manual.tex

```diff
@@ -5365,6 +5365,91 @@ \subsubsection{Tracking finite strain}
+  work with geomIO or cause errors. Moreover, geomIO currently does not support grouping paths (paths can still 
+  be combined using \texttt{Path$\rightarrow$Union}, \texttt{Path$\rightarrow$Difference} or similar commands), 
+  and only the outermost closed contour of a path will be considered. This means that i.e. for modelling a spherical 
+  annulus, you would have to draw two circles, and assign the inner one the same phase as the background of your drawing.}
+
+We will here use a drawing of a jellyfish located in \url{doc/manual/cookbooks/geomio/jellyfish.svg}, where different phases 
+have already been assigned to each path (Figure \ref{fig:jelly-picture}). 
+\begin{figure}[h!]
+    \centering
+    \includegraphics[width=0.2\textwidth]{cookbooks/geomio/jellyfish.pdf}
+    \caption{Vector drawing of a jellyfish.}
+    \label{fig:jelly-picture}
+\end{figure}
+\note{The page of your drawing in Inkscape should already have the extents (in px) that you later want to use in your model (in m).}
+
+After geomIO is initialized in MATLAB, we \href{http://geomio-doc.bitbucket.org/tuto2D.html#assigning-phase-to-markers}
+{run geomIO as desribed in the documentation}, loading the default options and then specifying all the option we want to 
+change, such as the path to the Input file, or the resolution. 
+\lstinputlisting[language=matlab]{cookbooks/geomio/run_geomio.part1.m}
+You can view all of the options available by typing \texttt{opt}. 
+
+In the next step we create the grid that is used for the coordinates in the ascii data initial conditions file 
+and assign a phase to each grid point: 
+\lstinputlisting[language=matlab]{cookbooks/geomio/run_geomio.part2.m}
+You can plot the \texttt{Phase} variable in MATLAB to see if the drawing was read in and all phases are assigned correctly 
+(Figure \ref{fig:jelly-plot}). 
+\begin{figure}[h!]
+    \centering
+    \includegraphics[width=0.45\textwidth]{cookbooks/geomio/jelly.png}
+    \caption{Plot of the \texttt{Phase} variable in MATLAB.}
+    \label{fig:jelly-plot}
+\end{figure}
+Finally, we want to write output in a format that can be read in by \aspect{}'s \texttt{ascii data} compositional
+initial conditions plugin. We write the data into the file \texttt{jelly.txt}.
+\lstinputlisting[language=matlab]{cookbooks/geomio/save_file_as_txt.m}
+
+To read in the file we just created (a copy is located in \aspect{}'s data directory), 
+we set up a model with a box geometry with the same extents we specified for the drawing in px 
+and one compositional field. We choose the ascii data compositional initial conditions and specify that we 
+want to read in our jellyfish. The relevant parts of the input file are listed below:
+\lstinputlisting[language=prmfile]{cookbooks/geomio/geomIO.prm}
+
+If we look at the output in \texttt{paraview}, we can see our jellyfish, with the mesh refined at the 
+boundaries between the different phases (Figure \ref{fig:jelly-paraview}). 
+\begin{figure}[h!]
```

Comment:
same here.

One typically puts a tilde instead of a space in "Figure \ref..." because it looks odd to have the number by itself if latex decides to break the line between these two parts. The tilde is a no-break space.